pkg/plugins/policies/meshcircuitbreaker/plugin/xds/configurer.go

```diff
@@ -210,3 +220,29 @@ func configureTotalFailures(
 	outlierDetection.EnforcingConsecutive_5Xx = util_proto.UInt32(100)
 	outlierDetection.Consecutive_5Xx = util_proto.UInt32(*conf.Consecutive)
 }
+
+func configureHealthyPanicThreshold(cluster *envoy_cluster.Cluster, value *intstr.IntOrString) error {
+	if value == nil {
+		return nil
+	}
+	if cluster.CommonLbConfig == nil {
+		cluster.CommonLbConfig = &envoy_cluster.Cluster_CommonLbConfig{}
+	}
+	percentage, err := envoyPercent(*value)
+	if err != nil {
+		return err
+	}
+	cluster.CommonLbConfig.HealthyPanicThreshold = percentage
+	return nil
+}
+
+func envoyPercent(intOrStr intstr.IntOrString) (*envoy_type.Percent, error) {
```

Comment:
Can we move it to `pkg/util/proto/types.go` ? We have all the Envoy structures there

Reply:
I noticed that the `pkg/util/proto` path doesn't import any `github.com/envoyproxy/go-control-plane/envoy/*` dependencies. They import `google.golang.org/protobuf/types/*`

How about putting this function next to https://github.com/kumahq/kuma/blob/0411233912021f6b3392a8155bc0d321ee426ae0/pkg/xds/envoy/listeners/v3/util.go#L84

Reply:
I'm not sure since this is `listener` code and we configure the cluster. Seems like it should be in some common place

Reply:
I'll create a **chore** PR to deal with the `envoyPercent` function in the future once we find a proper place. (This private function exists in multiple places for different policies)